src/content/wiki/tutorials/getting-started.mdoc

```diff
@@ -123,13 +123,13 @@ Get enough materials and craft your Supply Ship/Camp, then place it by right-cli
 
 ### Step 2
 
-After you have carefully decided where you want to place your {% building name="townhall" /%} (remember, the position where you placed the {% building name="townhall" /%} block at first will be the center of your colony's protected area. Once placed, the area will be set and cannot be changed), use your {% item name="scan_tool" /%} to place the {% building name="townhall" /%}. Once you commit to the placement of the {% building name="townhall" /%} (the green checkmark), the {% building name="townhall" /%} block will be placed. To officially start a colony, you need to right-click on the block, then select Create New Colony. Once you've done this, your initial four citizens will start appearing! 
+After you have carefully decided where you want to place your {% building name="townhall" /%} (remember, the position where you placed the {% building name="townhall" /%} block at first will be the center of your colony's protected area. Once placed, the area will be set and cannot be changed), use your {% item name="build_tool" /%} to place the {% building name="townhall" /%}. Once you commit to the placement of the {% building name="townhall" /%} (the green checkmark), the {% building name="townhall" /%} block will be placed. To officially start a colony, you need to right-click on the block, then select Create New Colony. Once you've done this, your initial four citizens will start appearing! 
 
 You can now right-click the {% building name="townhall" /%} block again to look what else you can configure in your colony, like your colony name!
 
 ### Step 3
 
-Before any other building can be built, you must craft the {% building name="builder" /%} block and use your {% item name="scan_tool" /%} to place it. Once you commit to the placement of the Builder's Hut (the green checkmark), the block will be placed and a Builder will be automatically assigned (or you can manually assign one with the best [traits](/wiki/systems/worker) for a Builder if you changed this in the setting tab in the {% building name="townhall" /%} its GUI).
+Before any other building can be built, you must craft the {% building name="builder's hut" /%} block and use your {% item name="build_tool" /%} to place it. Once you commit to the placement of the Builder's Hut (the green checkmark), the block will be placed and a Builder will be automatically assigned (or you can manually assign one with the best [traits](/wiki/systems/worker) for a Builder if you changed this in the setting tab in the {% building name="townhall" /%} its GUI).
```

Comment:
`{% building %}` tags rely on building ids, these are not display names, so you cannot say `{% building name="builder's hut" /%}` as this needs the building id (builder)